webpack/envs/baseConfig.js

```diff
@@ -61,7 +66,22 @@ exports.baseConfig = {
         NODE_ENV: JSON.stringify(NODE_ENV),
       },
     }),
-    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/), // Remove moment.js localization files
+    // Remove moment.js localization files
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    // Remove server-only modules from client bundles
+    ...(BUILD_SERVER
+      ? []
+      : [
+          // Remove server side of relay network layer.
+          new webpack.IgnorePlugin(
+            /^react-relay-network-modern-ssr\/node8\/server/
+          ),
+          // No matter what, we don't want the graphql-js package in client
+          // bundles. This /may/ lead to a broken build when e.g. a reaction
+          // module that's used on the client side imports something from
+          // graphql-js, but that's better than silently including this.
+          new webpack.IgnorePlugin(/^graphql(\/.*)?$/),
```

Comment:
> This /may/ lead to a broken build when e.g. a reaction module that's used on the client side imports something from graphql-js, but that's better than silently including this.

Is it better? Or does it just give us more stuff to have to mentally track and be aware of? What is the problem with this package?

Reply:
It’s a large package and absolutely unnecessary. In lieu of a setup where through ES module tree-shaking these things are automatically excluded and code in Reaction already having been architected so far along that it is hard to better split server vs client code, this is the next best thing.

Reply:
Over time it will be known that people shouldn’t import from such packages in code that gets used in clients, but that’s a matter of socializing through PR reviews.

Reply:
I see, so the problem is that `graphql-js` is a server-side-only package; this wasn't clear by the PR or the ticket linked above. Maybe it can be added to the comment?